machines/Makefile

```diff
@@ -2,7 +2,7 @@
 
 BSG_MANYCORE_DIR := $(shell git rev-parse --show-toplevel)
 # By convention, basejump_stl is in the same directory as $(BSG_MANYCORE_DIR)
-BASEJUMP_STL_DIR := $(BSG_MANYCORE_DIR)/../basejump_stl
+BASEJUMP_STL_DIR := $(abspath $(BSG_MANYCORE_DIR)/../basejump_stl)
```

Comment:
Out of curiosity, why was this necessary? (I like the change)

Reply:
Just for aesthetic change.

Reply:
Cool.